Python/flowgraph.c

```diff
@@ -23,6 +23,11 @@
         return ERROR;       \
     }
 
+#define RETURN_IF_NOT_CONST_SEQ(X)  \
```

Comment:
This macro implements "return if NULL", not "return if it's not a const sequence".